src/main/kotlin/step2/calculator/Calculator.kt

```diff
@@ -0,0 +1,34 @@
+package step2.calculator
+
+import step2.calculator.StringExpression.parseToMathExpression
+
+object Calculator {
+    private const val ARITHMETIC_EXCEPTION_ERROR_MESSAGE = "Error occurred during calculate"
+
+    fun calculate(expression: String): Int {
+        val mathExpression = parseToMathExpression(expression)
+
+        return kotlin.runCatching {
+            calculate(mathExpression)
+        }.onFailure {
+            throw ArithmeticException(ARITHMETIC_EXCEPTION_ERROR_MESSAGE)
+        }.getOrThrow()
+    }
+
+    private fun calculate(mathExpression: MathExpression): Int {
+        var result: Int = mathExpression.operands[0].value
+
+        for (i in 0 until mathExpression.operatorsSize) {
```

Comment:
```suggestion
        repeat(mathExpression.operatorsSize) {
```
repeat을 활용해도 좋을 것 같아요